configs/12.0/packages/tcmalloc/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="TCMalloc"
 ATSRC_PACKAGE_VER=2.7
-ATSRC_PACKAGE_REV=f47a52ce85c3
+ATSRC_PACKAGE_REV=9608fa3bcf80
```

Comment:
OK.

tcmalloc doesn't provide stable branches and this commit is the same as the tag:

```
$ git describe --tags 9608fa3bcf80
gperftools-2.7
```